lib/ddtrace/opentracing/carrier.rb

```diff
@@ -0,0 +1,6 @@
+module Datadog
+  module OpenTracing
+    class Carrier < ::OpenTracing::Carrier
```

Comment:
👍 the python opentracing library does not define a class for the carrier and is assumed to be a `dict`, this is better.